### PR TITLE
update mfs.cgi help | iframe --> docs.lizardfs.org

### PR DESCRIPTION
**Pull request by jwest75674**
Mostly just learning how to use Github. I manually make this change when updating lizardfs.

Simply adding an iframe to the docs onto the help page of mfscgi, which is currently mostly unused.

### CODE REVIEW

File: src/cgi/mfs.cgi.in [hunk 1]
```diff
@@ -3219,6 +3219,7 @@ if "HELP" in sectionset:
     # print_file("@CGI_PATH@/help.html")
     print("""please contact with help@lizardfs.com""")
     print("""<br/>""")
+    print """<iframe src="https://docs.lizardfs.com/", width="100%", height="100%"></iframe>"""
```
Comment (zserik) [inline src/cgi/mfs.cgi.in:3222]:
```suggestion
    print("""<iframe src="https://docs.lizardfs.com/", width="100%", height="100%"></iframe>""")
```

Comment (jwest75674) [reply src/cgi/mfs.cgi.in]: You're absolutely correct. My Spot on, my apologies in that regard.

Thanks for your patience, that was a novice mistake.

**Comment by zserik** [comment]
I think you should squash your commits, especially because you only change 1 line of code...

**Comment by jwest75674** [comment]
Thanks, It was one of my first commits in GitHub, just trying to learn the ropes. 

Will be sure to avoid single line commits in future.

File: src/cgi/mfs.cgi.in [hunk 2]
```diff
@@ -3219,6 +3219,7 @@ if "HELP" in sectionset:
     # print_file("@CGI_PATH@/help.html")
     print("""please contact with help@lizardfs.com""")
     print("""<br/>""")
+    print("""<iframe src="https://docs.lizardfs.com/", width="100%", height="100%"></iframe>""")
```
Comment (zserik) [inline src/cgi/mfs.cgi.in:3222]:
```suggestion
    print("""<iframe src="https://docs.lizardfs.com/" width="100%" height="100%"></iframe>""")
```
I just noticed that you have another mistake in your code: `,` aren't used to separate HTML attributes.

**Comment by jwest75674** [comment]
Hey, Sorry to be a pain, finally have some time to dive back into this:
Not sure what I've done on my local side, but going to close this and try again shortly (with more experience)

**Pull request closed without merging**